webapp/public/custom.css

```diff
@@ -220,4 +220,33 @@ div.blocklyWidgetDiv {
     position: relative !important;
     display: inline-block;
     float: right;
+}
+
+/*******************************
+    Light Mode
+*******************************/
+
+.light * {
+    /*CSS transitions*/
+  -webkit-transition: none!important;
+  -moz-transition: none!important;
+  -ms-transition: none!important;
+  transition: none!important;
+}
+
+.light .ui.dimmer {
+    animation-duration: 0s !important;
+}
+
+.light .ui.loader:after {
+    animation: none !important;
+}
+
+/* Remove blockly filter on drag */
+.light svg.blocklyBlockDragSurface > g {
+    filter: none !important;
+}
+
+.light .blocklyMainBackground {
+    fill: transparent !important;
```

Comment:
should we have a solid background to speed things up?

Reply:
Yeah we could use the fill color for blocklySvg.